Makefile

```diff
@@ -166,6 +173,27 @@ wait-kubernetes-dependencies: ## Wait for dependencies to be ready
 	  --selector=app.kubernetes.io/component=controller \
 	  --timeout=90s
 
+update-version: ## update release version
+	find . -type f | xargs sed -i 's/$(VERSION)/${RELEASE_VERSION}/g'
+
+bump-version: ## update post-release version and update commit message
+	find . -type f | xargs sed -i 's/$(RELEASE_VERSION)/${NEXT_VERSION}/g'
+
+git-commit: ## adds all files and bumps the version
+	git add .
```

Comment:
May be a safer choice, use `git add -u .`? So it will commit only tracked files

Reply:
Good point, at this time we do not expect any files which are untracked. Will fix.